src/service/receipt.service.ts

```diff
@@ -0,0 +1,79 @@
+
+import {
+    ArtifactExpiryReceipt,
+    BalanceChangeReceipt,
+    BalanceTransferReceipt,
+    InflationReceipt,
+} from 'nem2-sdk';
+
+export class ReceiptService {
+    constructor() {
+
+    }
+
+    /**
+     * @description The logic of formatting Receipt info
+     * @param receipt
+     * @returns {string}
+     */
+    public formatReceiptToFilter(statement: any): string {
+        let txt = '';
+        if (statement.transactionStatements[0]) {
```

Comment:
Please, render as well addressResolutionStatements and mosaicResolutionStatements: https://nemtech.github.io/nem2-sdk-typescript-javascript/classes/_model_receipt_statement_.statement.html